src/org/usfirst/frc/team467/robot/DriverStation.java

```diff
@@ -98,7 +98,7 @@ public void periodic() {
 	 */
 	public DriveMode getDriveMode() {
 		// TODO: Set the drive mode based on the buttons pushed
-		return DriveMode.ArcadeDrive; // Update with the correct drive mode
+		return DriveMode.CurvatureDrive; // Update with the correct drive mode
```

Comment:
Did we want to keep this? I don't remember if we finally decided we liked it.

Reply:
we don't know yet, more research needs to be done on CurvatureDrive, but it won't be to hard to switch the drive mode in the future. If I'm not mistaken, Bryan wants to do more testing on this Drive mode too.